web/src/components/pages/datasets/stats.ts

```diff
@@ -38,7 +38,7 @@ export default {
           eighties: 0,
           nineties: 0,
         },
-        gender: { '': 0.39, male: 0.46, female: 0.13, other: 0.02 },
+        sex: { '': 0.39, male: 0.46, female: 0.13, other: 0.02 },
```

Comment:
This is something we'll have to discuss, because these labels are also used in our dataset, and changing a column name is a breaking change as far as dataset continuity is concerned and right now if we changed it outright it would break DeepSpeech. We can either ask them to conform to our new label (and I don't know if there are industry standards regarding this) or handle the label translation in our bundler code (no input from ML team needed). Any thoughts on this @mbransn?

Reply:
If I'm reading this correctly, the proposal is to move to `sex` from `gender` as this column label? If yes, I'm surprised this wasn't already the case in the dataset since `sex` is what is utilized across the platform. This was an intentional choice based on industry standard for articulating sex in datasets (and I believe Kelly was part of that discussion with Henretty at the time?). At minimum let's handle on the bundler side and ideally work for uniformity with DS.